juniper_official/interface/get-optical-interface-parameters.rule

```diff
@@ -0,0 +1,94 @@
+/*
+ * Collect Optical Interface Parameters:
+ *      1) Interface Name
+ *      2) Lane index
+ *      3) Lane Laser Bias Current
+ *      4) Lane Laser Output Power
+ *      5) Lane Laser Receive Power
+ */
+
+healthbot {
+    topic interfaces {
+        rule get-optical-interface-parameters {
+            keys [ interface-name lane-index];
+            synopsis "Rule for Optics Anomaly Detection";
+            description "This rule collects optical interface parameters";
+            sensor sensor_optics {
+                open-config {
+                    sensor-name /junos/system/linecard/optics/;
+                    frequency 60s;
+                }
+            }
+
+            field interface-name {
+                sensor sensor_optics {
+                    where "/interfaces/interface/@name =~ /{{interface-name}}/";
+                    path "/interfaces/interface/@name";
+                }
+                type string;
+            }
+
+            field lane-index {
+                sensor sensor_optics {
+                    path "/interfaces/interface/optics/lanediags/lane/@number";
+                }
+                type string;
+                description "Interface lane index";
+            }
+
+            field lane-laser-bias-current {
+                sensor sensor_optics {
+                    path /interfaces/interface/optics/lanediags/lane/lane_laser_bias_current;
```

Comment:
Only this path is missing in https://github.com/Juniper/healthbot-rules/blob/jcloud/juniper_official/interface/check-optical-signal-loss-fec-tx-rx-power.rule. We should enhance this rule instead of writing a new one.